### ISSUE TITLE
Some non-standard WCS keywords were excluded

### ISSUE THREAD

**Issue by pllim** [spacetelescope/drizzlepac#18]
Example test: https://ssb.stsci.edu/pandokia/pandokia.cgi?query=detail&key_id=36464239

```
[WARNING ] Some non-standard WCS keywords were excluded:
A_3_1, A_3_0, B_3_0, B_3_1, B_1_2, B_1_3, B_1_1, B_2_1, B_2_0, B_2_2,
B_ORDER, A_ORDER, B_0_4, B_0_3, B_0_2, A_4_0, B_4_0,
A_1_1, A_1_3, A_1_2, A_0_4, A_0_2, A_0_3, A_2_2, A_2_0, A_2_1
Use the ``relax`` kwarg to control this.
```

**Comment by mcara** [comment]
This issue and #19 are related and these warnings are coming not from drizzlepac itself but rather from ``stwcs.updatewcs.updatewcs()``. For example, in the ``acs/wfc/acs_tweak`` directory run:
```python
>>> from stwcs.updatewcs import updatewcs

>>> updatewcs('*_flt.fits')
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 6 for chip 2 : F606W
WARNING: Some non-standard WCS keywords were excluded: A_3_1, A_3_0, B_3_0, B_3_1, B_1_2, B_1_3, B_1_1, B_2_1, B_2_0, B_2_2, B_ORDER, A_ORDER, B_0_4, B_0_3, B_0_2, A_4_0, B_4_0, A_1_1, A_1_3, A_1_2, A_0_4, A_0_2, A_0_3, A_2_2, A_2_0, A_2_1 Use the ``relax`` kwarg to control this. [astropy.wcs.wcs]
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 6 for chip 2 : F606W
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 5 for chip 1 : F606W
WARNING: VerifyWarning: Card is too long, comment will be truncated. [astropy.io.fits.card]
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 6 for chip 2 : F606W
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 6 for chip 2 : F606W
TDDORDER kw not present, using default TDD correction
- IDCTAB: Distortion model from row 5 for chip 1 : F606W
['j94f05bgq_flt.fits', 'j9irw1rqq_flt.fits']
```

The two warnings above are exactly the ones described in this issue and issue #19. I do not think there is anything that can be done in drizzlepac about this.

**Comment by mcara** [comment]
CC: @nden @stsci-hack

**Comment by mcara** [comment]
@stsci-hack @nden Upon further investigation, this warning appears to be due to a mess in the WCS of the input test files. For example, in ``stis/stis_04``, ``CTYPE`` does not have ``-SIP`` postfix but do contain SIP coefficients (why? how???). In ``acs\wfc\acs_tweak`` input input images to ``updatewcs`` have both ``-SIP`` in ``CTYPE`` and SIP coefficients but the original WCS does not have ``-SIP`` postfix. So, I think this should be solved by cleaning the WCS of the input images @stsci-hack

**Comment by mcara** [comment]
@pllim @stsci-hack Is this issue still relevant today? I suspect that the new tests would not have these warnings. Can we close this issue?

**Comment by pllim** [comment]
Yeah, sure. Any warnings can be addressed as part of #150. Fun, fun!
